example/index.html

```diff
@@ -23,7 +26,7 @@ <h2 id="multiline-text">Multiline text copy</h2>
 some multiline text
 for us to copy</textarea>
     <div class="half">
-      <button data-test="init-multiline-text">Click to copy multiline text</button>
```

Comment:
I thought that `data-test` wasn't used at all so I repurposed this to make it easier to target buttons to add the click event listeners.